app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find_by_id(params[:id])
+  end
+end
```

Comment:
**OPTIONAL**- Though these controllers work fine, I suggest that you have no logic in your controller yet. Please fix this 🙏